scripts/orderlywebconfig.properties

```diff
@@ -0,0 +1,6 @@
+app.url=https://localhost/reports
+auth.fine_grained=false
+montagu.api_url=https://172.17.0.1/api/v1
```

Comment:
This is needed because 'localhost' from inside the container will only access the container. This is the static IP for the docker network's host.

Reply:
I think the existing `http://api:8080` will also work!